Python/aot_ceval_jit.c

```diff
@@ -3998,24 +4085,22 @@ void* jit_func(PyCodeObject* co, PyThreadState* tstate) {
         case UNARY_NOT:
         case UNARY_INVERT:
         case GET_ITER:
+#if PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION >= 9
+        case LIST_TO_TUPLE:
+#endif
         {
             RefStatus ref_status = deferred_vs_pop1(Dst, arg1_idx);
             deferred_vs_convert_reg_to_stack(Dst);
-            void* func = get_aot_func_addr(Dst, opcode, oparg, 0 /*= no op cache */);
+            void* func = NULL;
+#if PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION >= 9
+            if (opcode == LIST_TO_TUPLE) {
+                func = PyList_AsTuple;
+            } else
+#endif
+            func = get_aot_func_addr(Dst, opcode, oparg, 0 /*= no op cache */);
```

Comment:
Would you mind adding braces around this line? Or something to make it a bit more clear that this is (sometimes) the body of the else block.